src/components/MetadataEditor.vue

```diff
@@ -2,10 +2,6 @@
   <div id="md-editor-container">
     <div style="position: relative;">
       <div id="md-editor-submit">
-        <router-link :to="opticalImageAlignmentHref" v-if="datasetId"
```

Comment:
the `opticalImageAlignmentHref` property should be removed as well from the component